### ISSUE TITLE
[Enhancement] User Guide

### ISSUE THREAD

**Issue by fohfuu** [AnySoftKeyboard/AnySoftKeyboard#3218]
I downloaded the app today, and it seems really customisable, but it's very difficult for me to find all the settings that I want to change. There are when Tweak settings nested in other Tweak settings. There doesn't appear to be much help for end users on this. Any documentation on how to find particular settings or what they do would be useful, for example:

- A page on the website, possibly integrated into the app

- a Wiki on the Github

- a Search function, similar to the one found in modern OS settings

However, there are many options I have not considered.

If this already exists somewhere, I request this is made easier to find on the app or website. Thanks!

**Comment by mamg22** [comment]
I agree, the ways settings are distributed is really messy. I really like AnySoftKeyboard, it's the only one that covers my needs and it has a good design, except for the configuration pages, I find myself lost everytime I need to change a setting.

As an example, [this reddit thread](https://www.reddit.com/r/fossdroid/comments/sjyp63/looking_for_foss_keyboard_with_split_keyboard/) where users thought there wasn't a split keyboard option in ASK because it is quite hard to find. I mean, it's under a "Tweaks" subsection inside a "Tweaks & More" section, how would I know is there?

I think one of the best things that can be done about this is splitting and reordering all the settings and put them into smaller sections of related settings, instead of cramming everything into a generic section of "Tweaks".

**Comment by alchemiker** [comment]
Yeeeees! 🙌 
I suggested this already a while ago (https://github.com/AnySoftKeyboard/AnySoftKeyboard/discussions/2802) but with no result. And it was also suggested (https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/3047) by @dagelf (https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/3047) too. 
Since no productive answers or results... : (

**Comment by menny** [comment]
I'm really sorry about this mess, it's hard to keep this project updated with so many things happening.
I replied to https://github.com/AnySoftKeyboard/AnySoftKeyboard/discussions/2802
